kbase-extension/static/kbase/custom/custom.js

```diff
@@ -145,19 +143,8 @@ define([
             options = defaultValue;
             defaultValue = undefined;
         }
-        let missing = false;
```

Comment:
This hunk of code here tests if the string passed in for processing is a number in various ways. We're using the [numeral](http://numeraljs.com/) package here, so just let it tell us. Also, the real bug was using `isNaN` on the `numeralValue` object, which was erroring. Not sure why this took so long to fail / be found.

Reply:
I'm sure it's the only unfound bug in the codebase!